aiida_quantumespresso/parsers/parse_raw/cp.py

```diff
@@ -81,54 +88,56 @@ def parse_cp_text_output(data, xml_data):
                 parsed_data['wall_time'] = time
             except:
                 raise QEOutputParsingError('Error while parsing wall time.')
-
-    for count, line in enumerate(reversed(data)):
-        if 'nfi' in line and 'ekinc' in line and 'econs' in line:
-            this_line = data[len(data) - count]
-            try:
-                parsed_data['ekinc'] = [float(this_line.split()[1])]
-            except ValueError:
-                pass
-            try:
-                parsed_data['temph'] = [float(this_line.split()[2])]
-            except ValueError:
-                pass
-            try:
-                parsed_data['tempp'] = [float(this_line.split()[3])]
-            except ValueError:
-                pass
-            try:
-                parsed_data['etot'] = [float(this_line.split()[4])]
-            except ValueError:
-                pass
-            try:
-                parsed_data['enthal'] = [float(this_line.split()[5])]
-            except ValueError:
-                pass
-            try:
-                parsed_data['econs'] = [float(this_line.split()[6])]
-            except ValueError:
-                pass
-            try:
-                parsed_data['econt'] = [float(this_line.split()[7])]
-            except ValueError:
-                pass
-            try:
-                parsed_data['vnhh'] = [float(this_line.split()[8])]
-            except (ValueError, IndexError):
-                pass
-            try:
-                parsed_data['xnhh0'] = [float(this_line.split()[9])]
-            except (ValueError, IndexError):
-                pass
-            try:
-                parsed_data['vnhp'] = [float(this_line.split()[10])]
-            except (ValueError, IndexError):
-                pass
-            try:
-                parsed_data['xnhp0'] = [float(this_line.split()[11])]
-            except (ValueError, IndexError):
-                pass
+    try:
```

Comment:
Is this still needed, and when? If you have a reproducible case, maybe we should see if we can skip according to a flag, or put some message in the parsed output to say we skipped for reason XXX

Reply:
The parser works only for the verlet output. When the code is doing cg steps, the output is different, and maybe we need a new parser for that